src/plugins/telemetryTable/TelemetryTable.js

```diff
@@ -215,8 +215,12 @@ define([
           return;
         }
 
+        const configuration = this.telemetryObjects[keyString].telemetryObject.configuration;
+        const updateInPlace = configuration && configuration.inPlaceUpdates === true;
```

Comment:
I like this approach. I wonder if it might be even better suited to telemetry metadata, which is where we normally provide this sort of hint to telemetry views about how to process the incoming telemetry?

So in the case of command history the [command history object provider](https://github.com/akhenry/openmct-yamcs/pull/323/files#diff-0782d46d31b484394d9df9502c2035215827dedeab58f6bb6840cf2d3b31798eR44) might provide the relevant metadata like so:

```
...
        telemetry: {
            values: [
                {
                  key: 'id',
                  name: 'row identifier',
                  useToUpdateInPlace: true
                }
```

This would involve adding the `id` to the telemetry metadata for command history, which it isn't right now, but I think that's fine.

This allows the developer to easily choose which key to match on, rather than hard-coding `messageId`. It also potentially allows for matching on composite keys in future.

Reply:
Done.